Add `Announcement messages` (#9054)
ppy/osu-wiki: wiki/Announcement_messages/en.md

---
tags:
  - announce
  - announce usergroup
  - announce user group
---

# Announcement messages

![Announcement notification](img/notification.jpg "An announcement message notification")

An **announcement message** is a special type of message that's intended for sending longer and formatted messages to multiple users at once. The key differences between announcement messages and regular chat messages are:

> - The 1024 character limit instead of 450

Author: i asked about this in #osu-web and [this](https://discord.com/channels/188630481301012481/188630616286167041/1068380206122934372) is the response i was redirected to

![image](https://user-images.githubusercontent.com/62819481/228686987-7f71c229-d911-4a73-990b-c907181595d9.png)

> - Announcement messages have a 1024 character limit instead of 450.
> - Announcement messages support Markdown[^note-images] syntax for text formatting.
> - Announcement messages are sent to all users in the specified channel.
> - Announcement messages bypass the `block private messages from people not on your friends list` setting.
> - Normal users cannot reply to announcement messages.

Reviewer: making it less repetitive
also, 3) isn't really a difference, since regular chat messages are visible by all channel users as well
```suggestion
- The 1024 character limit instead of 450
- Markdown syntax support[^note-images] for text formatting
- Delivery to all users in the specified channel
- Ability to bypass the `block private messages from people not on your friends list` setting.
- Normal users cannot reply to announcement messages.
```

Author: i'll rename it to something like `Delivery to multiple users at once`

## Eligibility

> In order to send and reply to announcement messages through the website, you must be a member of the [Global Moderation Team](/wiki/People/Global_Moderation_Team), the [Nomination Assessment Team](/wiki/People/Nomination_Assessment_Team), or the announce [user group](/wiki/People/User_group). However, only members of the announce user group are allowed to send chat announcements through the [osu! API v2](https://docs.ppy.sh).

Reviewer 2: API operates on a different permission set than the regular parts of the website, which is why you need to be in ``announce`` usergroup to use it, no matter if you are a GMT/NAT or not.

You will get a ``Client error: `POST http://osu.ppy.sh/api/v2/chat/channels` resulted in a `403 Forbidden` response: {"error":"This channel is for announcements only."}`` if you try without ``announce`` (tested).

### Filing a request

You can file a request to join the announce user group by sending an email to [accounts@ppy.sh](mailto:accounts@ppy.sh) with the subject `Announce Usergroup Request`. This must be sent from the email address attached to your osu! account.

The body of the email should contain the following:

- Your osu! username
- An explanation outlining your reason for needing announcement messages, and how frequently you will be using them.

> The [account support team](/wiki/People/Account_support_team) will review your request. If your request is denied, they will let you know why. If your request is approved, they will add you to the announce user group and inform you of this.

Reviewer: ```suggestion
The [account support team](/wiki/People/Account_support_team) will review the request and inform you of their decision.
```

## Sending announcement messages

> In order to send a chat announcement, open the [chat page](https://osu.ppy.sh/community/chat) from your chat notifications and click the `create announcement` button. Input the channel name, description, list of recipients, and your message. Finally, click the `create` button to send the announcement.

Author: will mention with a footnote

> In order to send a chat announcement, open the [chat page](https://osu.ppy.sh/community/chat) from your chat notifications tab. Then, click the `create announcement` button. this will open a dedicated page for creating announcements. Input the channel name, description, list of users to message, then your message. Finally, click the `create` button to send the announcement.

Reviewer: ```suggestion
In order to send a chat announcement, open the [chat page](https://osu.ppy.sh/community/chat) from your chat notifications and click the `create announcement` button. Input the channel name, description, list of recipients, and your message. Finally, click the `create` button to send the announcement.
```

![Announcement creation page](img/page.jpg "The announcement creation page")

## Trivia

> - Announcement messages are intended to be a direct replacement for old forum messages.

Reviewer: ```suggestion
- Announcement messages are intended to be a direct replacement for old [forum](/wiki/Community/Forum) messages.
```

> - [Basic implementation](https://github.com/ppy/osu-web/pull/8418) of the announcement system was added to the website January 26th, 2022. This included the announce user group and the ability to send announcement messages through the API. The user interface for sending chat announcements, alongside allowing moderators to send them, was [added](https://github.com/ppy/osu-web/pull/8747) on June 1st, 2022.

Reviewer: ```suggestion
- [Basic implementation](https://github.com/ppy/osu-web/pull/8418) of the announcement system was added to the website on January 26th, 2022. This included the announce user group and the ability to send announcement messages through the API. The user interface for sending chat announcements, alongside allowing moderators to send them, was [added](https://github.com/ppy/osu-web/pull/8747) on June 1st, 2022.
```

> - The announce user group's ID is 47, it has no group badge nor a colour, and its user listing is not public.

Reviewer: ```suggestion
- The announce user group's ID is 47, it has neither a group badge nor a dedicated colour, and its user listing is private.
```
